src/components/ResultsCount/ResultsCount.tsx

```diff
@@ -32,7 +32,7 @@ export const ResultsCount = ({
 
   return (
     <div className={classNames}>
-      <p className="cc-results-count__result">
+      <p className="cc-results-count__result" aria-live="polite" role="alert">
```

Comment:
Is the use of `role="alert"` here appropriate? From [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role):

> Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required. Dynamic changes that are less urgent should use a less aggressive method, such as aria-live="polite" or other live region roles.

Reply:
As I understand it, adding `role="alert"` implies that an element will be assertive (the same as adding `aria-live="assertive"` to the element). If `aria-live="polite"` is used the screen reader will wait until after it has finished its current thread before announcing the alert.

Reply:
☝️  it was in session one of the a11y training yesterday :)

Reply:
Thanks Chris. re: assertive, I can see that's mentioned in the notes from the MDN docs: 

> Using the alert role on an element implies that element has aria-live="assertive".

My concern is whether aria-live="polite" and role="alert" are conflicting. There's no mention of using the combination in the MDN docs, but there is this line:

> Dynamic changes that are less urgent should use a less aggressive method, such as aria-live="polite" or other live region roles. 

Could either of you suggest any other docs I could take a look at regarding aria-live & role="alert"?